src/index.js

```diff
@@ -1,35 +1,42 @@
-import { scores } from './scorelog';
+import { initUsers, pullData, newGame } from './api';
 import './styles.css';
 
-const scorelog = [
-  {
-    name: 'Name',
-    score: 100,
-  },
-  {
-    name: 'Name',
-    score: 20,
-  },
-  {
-    name: 'Name',
-    score: 50,
-  },
-  {
-    name: 'Name',
-    score: 78,
-  },
-  {
-    name: 'Name',
-    score: 125,
-  },
-  {
-    name: 'Name',
-    score: 77,
-  },
-  {
-    name: 'Name',
-    score: 42,
-  },
-];
-
-document.addEventListener('DOMContentLoaded', () => scores(scorelog));
\ No newline at end of file
+const loadScores = async () => {
+  const displayScores = document.getElementById('scores');
+
+  while (displayScores.firstChild) {
+    displayScores.removeChild(displayScores.firstChild);
+  }
+
+  const userData = await pullData();
```

Comment:
- [ ] OPTIONAL: Ther is an uncaught type error on the console. While your app is functioning well it is important to fix this error or set error boundaries (error handling). :pray: 

![Screenshot from 2021-09-19 00-41-42](https://user-images.githubusercontent.com/64894927/133905462-43abbb68-df39-4ac3-8bf7-0976b39f2e4f.png)